docs/internals/contributing/writing-documentation.txt

```diff
@@ -258,6 +261,12 @@ documentation:
   also need to define a reference to the documentation for that environment
   variable using :rst:dir:`.. envvar:: <envvar>`.
 
+.. versionchanged:: 4.2
+
+    All code blocks in Django's docs was reformatted with `blacken-docs`_.
```

Comment:
```
    All Python code blocks in the Django docs were reformatted with
    `blacken-docs`_.
```

Could go for `s/docs/documentation/` but I think the friendly voice is fine.

Reply:
Updated :+1: